cds_ils/importer/providers/cds/rules/base.py

```diff
@@ -195,13 +201,13 @@ def doc_type_mapping(val):
         if val_a:
             if _doc_type and _doc_type != val_a:
                 raise ManualImportRequired(
-                    subfield="a" "inconsistent doc type"
+                    subfield="a - " "inconsistent doc type"
```

Comment:
this was wrongly formatted before apparently. It should be
```
raise ManualImportRequired("Inconsistent doc type", subfield="a")
```
`"Inconsistent doc type"` is the message passed to the exception, while subfield was meant to be a kwarg, they were not supposed to be a joint string